app/controllers/users_controller.rb

```diff
@@ -1,13 +1,18 @@
 class UsersController < ApplicationController
   before_action :set_user, only: %i[ show liked feed followers following discover ]
+  before_action { authorize @user || User }
 
-  private
+  def followers
+    true
```

Comment:
Keep in mind you are in the controller here, I don't think `true` should be here.